### DISCUSSION TITLE
ARM64 Docker images for standalone-chrome?

### DISCUSSION THREAD

**Discussion by huynguyen-anduin** [SeleniumHQ/docker-selenium#2379] [Q&A] [answered]
"Selenium Grid ARM64 Docker images now landing in the official [selenium] namespace on Docker Hub (https://lnkd.in/gmr_3PRd) from release tag 4.21.0."

ARM64 images for standalone-chrome not found: https://hub.docker.com/r/selenium/standalone-chrome/tags
Other browser is supported, such as firefox: https://hub.docker.com/r/selenium/standalone-firefox/tags

Is it missing or not supported?

**Comment by diemol** [comment, marked as answer]
Google does not provide a ARM binary for Chrome on Linux, hence we can only offer Chromium.
